map/framework.cpp

```diff
@@ -501,10 +501,10 @@ booking::Api const * Framework::GetBookingApi(platform::NetworkPolicy const & po
   return nullptr;
 }
 
-uber::Api * Framework::GetUberApi(platform::NetworkPolicy const & policy)
+taxi::Engine * Framework::GetTaxiEngine(platform::NetworkPolicy const & policy)
 {
   if (policy.CanUse())
-    return m_uberApi.get();
+    return m_taxiEngine.get();
```

Comment:
Please, check here that m_taxiEngine is initialized. We spent ~2 hours during hackaton investigating null-dereference, due to optimizing compilers null-dereference-bugs may be tricky, especially when they happen during JNI calls.